e2e/atlas/clusters_test.go

```diff
@@ -20,22 +20,20 @@ import (
 	"fmt"
 	"os"
 	"os/exec"
-	"strings"
 	"testing"
 
 	"github.com/mongodb/mongocli/e2e"
+	"github.com/stretchr/testify/assert"
 	"go.mongodb.org/atlas/mongodbatlas"
 )
 
 func TestClustersFlags(t *testing.T) {
-	cliPath, e := e2e.Bin()
-	if e != nil {
-		t.Fatalf("unexpected error: %v", e)
-	}
-	clusterName, e := RandClusterName()
-	if e != nil {
-		t.Fatalf("unexpected error: %v", e)
-	}
+	cliPath, err := e2e.Bin()
+	a := assert.New(t)
+	a.NoError(err)
+	clusterName, err := RandClusterName()
+	a.NoError(err)
```

Comment:
I'm ok if you want to keep migrating tests to use assert but want thing you need to be aware is that this change is not really equivalent and may introduce weird tests on error if you see the original test had a `t.Fatalf` which means if there was an error the rest of the test would not run, this is important as getting some errors should prevent further testing given they will fail either way.

With your change now if there's an error on line 33, line 34 is still being check, please consider making sure that if we have irrecuperable errors we actually finish the test early, there are examples around our e2e tests using `ifs` to do this